src/main/scala/it/pagopa/interop/tenantmanagement/model/persistence/Adapters.scala

```diff
@@ -164,14 +174,20 @@ object Adapters {
   }
 
   implicit class PersistentTenantMailWrapper(private val ptm: PersistentTenantMail) extends AnyVal {
-    def toApi: Mail = Mail(kind = ptm.kind.toApi, address = ptm.address, createdAt = ptm.createdAt)
+    def toApi: Mail = Mail(
+      kind = ptm.kind.toApi,
+      address = ptm.address,
+      createdAt = ptm.createdAt,
+      description = Option(ptm.description).filter(_.nonEmpty)
```

Comment:
any reason why the description is mandatory in persistence but optional in the api?

Reply:
Of course there is! And it's a pretty valid reason too: I made a mistake.

Reply:
🤣